utils/k8sutils.go

```diff
@@ -4,46 +4,48 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/rs/zerolog/log"
 	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
 )
 
-func GetK8sEventAttrs(client *CachedK8sClient, srcIp string, dstIp string) map[string]any {
-	log.Debug().
-		Str("src_ip", srcIp).
-		Str("dst_ip", dstIp).
-		Msg("Getting k8s event attrs")
+// GetK8sAttrsForIp returns a map of kubernetes metadata attributes for a given IP address.
+//
+// Provide a prefix to prepend to the attribute names, example: "source" or "destination".
+//
+// If the IP address is not found in the kubernetes cache, an empty map is returned.
+func GetK8sAttrsForIp(client *CachedK8sClient, ip string, prefix string) map[string]any {
```

Comment:
How about offer two wrapping funcs, `GetK8sAttrsForSourceIP` and `GetK8sAttrsForDestinationIP`, which include the "source" and "destination" prefixes?

It would simplify users to just call the correct func instead of knowing what prefix to use.

Reply:
Added in 3dcf19a. Whad'ya think?